lib/editors.js

```diff
@@ -1751,7 +1751,7 @@ class FileUploadEditor extends Editor {
       let err = new Error('Invalid file path');
       err.info =
         `<p>The file path</p>` +
-        `<div class="container"><pre class="bg-dark text-white rounded p-2">${this.oldPath}</pre></div>` +
+        `<div class="container"><pre class="bg-dark text-white rounded p-2">${this.filePath}</pre></div>` +
```

Comment:
👀 was this a bug?

Reply:
Yup. I was testing contains with an invalid path and this showed as undefined. I didn't do an extensive search for more bugs, only fixed the one I found. That said, this error is not expected to show during regular use, only if invalid paths are specified, so it's not a bug that regular users would experience.